evm/its.js

```diff
@@ -938,7 +967,7 @@ if (require.main === module) {
         .addArgument(new Argument('<type>', 'Token manager type').choices(Object.keys(tokenManagerTypes)))
         .argument('<operator>', 'Operator address')
         .addOption(new Option('--rawSalt <rawSalt>', 'raw deployment salt').env('RAW_SALT'))
-        .addOption(new Option('--gasValue <gasValue>', 'gas value').default(0))
+        .addOption(new Option('--gasValue <gasValue>', 'gas value'))
```

Comment:
```suggestion
        .addOption(new Option('--gasValue <gasValue>', 'gas value').default('auto'))
```
use an explicit default to distinguish from `0` and other values that are provided